src/libraries/Microsoft.Win32.Primitives/src/System/ComponentModel/Win32Exception.Default.cs

```diff
@@ -6,6 +6,6 @@ namespace System.ComponentModel
 {
     public partial class Win32Exception
     {
-        private static string GetErrorMessage(int error) => Interop.Sys.StrError(error);
+        private static string GetErrorMessage(int error) => "Unsupported platform";
```

Comment:
This will affect Unix non WASM right? I may be missing context, but there are cases where in Unix implementation we throw a Win32Exception using a Unix error code expecting StrError to make something nice of it.

https://github.com/dotnet/runtime/blob/106cd9ebaed5f974fa8f17e153ec3e22b6576f43/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs#L484

there are also cases where we try to use Windows error codes, which I doubt StrError is going to decode correctly, eg.,

https://github.com/dotnet/runtime/blob/106cd9ebaed5f974fa8f17e153ec3e22b6576f43/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs#L400

We ought to be consistent and as useful as possible, I think that probably means on actual Unix (not WASM) Win32Exception should be able to give you a nice message from StrError.

Reply:
Right, which is my comment here: https://github.com/dotnet/runtime/pull/37944#discussion_r449297812

We need both configurations.  Unix needs to stay for Unix.  We need to add a RID-less configuration with a default implementation of `GetErrorMessage` that returns a semi-useful string on Browser.